doc/rados/operations/health-checks.rst

```diff
@@ -28,162 +28,161 @@ Monitor
-available space on the file system storing the monitor database
-(normally ``/var/lib/ceph/mon``), as a percentage, drops below
+One or more monitors are low on disk space. This health check is raised if the
+percentage of available space on the file system used by the monitor database
+(normally ``/var/lib/ceph/mon``) drops below the percentage value
 ``mon_data_avail_warn`` (default: 30%).
 
-This may indicate that some other process or user on the system is
-filling up the same file system used by the monitor.  It may also
-indicate that the monitors database is large (see ``MON_DISK_BIG``
+This alert might indicate that some other process or user on the system is
+filling up the file system used by the monitor. It might also
+indicate that the monitor database is too large (see ``MON_DISK_BIG``
 below).
 
-If space cannot be freed, the monitor's data directory may need to be
-moved to another storage device or file system (while the monitor
-daemon is not running, of course).
+If space cannot be freed, the monitor's data directory might need to be
+moved to another storage device or file system (this relocation process must be carried out while the monitor
+daemon is not running).
 
 
 MON_DISK_CRIT
 _____________
 
-One or more monitors is critically low on disk space.  This alert
-triggers if the available space on the file system storing the monitor
-database (normally ``/var/lib/ceph/mon``), as a percentage, drops
-below ``mon_data_avail_crit`` (default: 5%).  See ``MON_DISK_LOW``, above.
+One or more monitors are critically low on disk space. This health check is raised if the
+percentage of available space on the file system used by the monitor database
+(normally ``/var/lib/ceph/mon``) drops below the percentage value
+``mon_data_avail_crit`` (default: 5%). See ``MON_DISK_LOW``, above.
 
 MON_DISK_BIG
 ____________
 
-The database size for one or more monitors is very large.  This alert
-triggers if the size of the monitor's database is larger than
+The database size for one or more monitors is very large. This health check
+is raised if the size of the monitor database is larger than
 ``mon_data_size_warn`` (default: 15 GiB).
 
-A large database is unusual, but may not necessarily indicate a
-problem.  Monitor databases may grow in size when there are placement
+A large database is unusual, but does not necessarily indicate a
+problem. Monitor databases might grow in size when there are placement
 groups that have not reached an ``active+clean`` state in a long time.
 
-This may also indicate that the monitor's database is not properly
-compacting, which has been observed with some older versions of
-leveldb and rocksdb.  Forcing a compaction with ``ceph daemon mon.<id>
-compact`` may shrink the on-disk size.
+This alert might also indicate that the monitor's database is not properly
+compacting, an issue that has been observed with some older versions of
+leveldb and rocksdb. Forcing a compaction with ``ceph daemon mon.<id>
```

Comment:
LevelDB and RocksDB. 

Forcing compaction with

Suggest adding a sentence about turning on `mon_compact_on_start`, which may work better than online compaction.

Reply:
unfinished business